tests/saw/spec/handshake/s2n_handshake_io.cry

```diff
@@ -257,8 +257,8 @@ state_machine_fn m =
   else zero
 
 // A model of the tls1.3 handshake state machine (array handshakes in C)
-tls13_handshakes : [512][32][32]
-tls13_handshakes = [tls13_handshakes_fn h | h <- [0..511]]
+tls13_handshakes : [1024][32][32]
```

Comment:
Did you have a way around this handshake growth? Or was that part of another PR?

Reply:
I've got ideas: https://github.com/aws/s2n-tls/pull/2628